Algorithm/CandlestickPatterns.cs

```diff
@@ -0,0 +1,508 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using System;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Indicators.CandlestickPatterns;
+
+namespace QuantConnect.Algorithm
+{
+    /// <summary>
+    /// Provides helpers for using candlestick patterns
+    /// </summary>
+    public class CandlestickPatterns
+    {
+        private readonly QCAlgorithm _algorithm;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CandlestickPatterns"/> class
+        /// </summary>
+        /// <param name="algorithm">The algorithm instance</param>
+        public CandlestickPatterns(QCAlgorithm algorithm)
+        {
+            _algorithm = algorithm;
+        }
+
+        /// <summary>
+        /// Creates a new Two Crows pattern indicator.
```

Comment:
We may want to make these `<see cref="TwoCrows" />` so that they can navigate to a description of the `TwoCrows` indicator, thoughts?